ArduPlane/Parameters.cpp

```diff
@@ -1314,7 +1313,6 @@ static const AP_Param::ConversionInfo conversion_table[] = {
 
     { Parameters::k_param_land_slope_recalc_shallow_threshold,0,AP_PARAM_FLOAT, "LAND_SLOPE_RCALC" },
     { Parameters::k_param_land_slope_recalc_steep_threshold_to_abort,0,AP_PARAM_FLOAT, "LAND_ABORT_DEG" },
-    { Parameters::k_param_land_pitch_cd,      0,      AP_PARAM_INT16, "LAND_PITCH_CD" },
```

Comment:
I wonder if we should just take the opportunity to remove all of these landing param conversions. With this change you most but not all. Looks like there 8 years old. https://github.com/ArduPilot/ardupilot/pull/5269

Reply:
separate PR so I don't need to retest for the 4.5 beta